test-requirements.txt

```diff
@@ -3,3 +3,4 @@ coverage
 nbformat
 pytest
 six
+attrs>=17.4.0 
```

Comment:
@ericdill even though we are using `pip` and [`jsonschema` is specifying the right `attrs`](https://github.com/Julian/jsonschema/blob/master/setup.cfg#L29) in the `setup.cfg` we still end up with the wrong version in Python 2.7 and 3.6. I can just ditch `pip` and use `conda` in the CIs or we can use this workaround until 2.7 EOL and someone fixes 3.6.